src/pages/UniversalSwap/Modals/SlippageModal.tsx

```diff
@@ -44,28 +44,31 @@ export const SlippageModal: FC<ModalProps> = ({ setUserSlippage, setVisible }) =
             {option}%
           </div>
         ))}
-        {/* <div
+        <div
           className={cx('item', 'border', {
             isChosen: indexChosenOption === OPTIONS_SLIPPAGE.length
           })}
           onClick={() => {
-            setUserSlippage(manualSlippage ?? 0);
             setIndexChosenOption(OPTIONS_SLIPPAGE.length);
           }}
         >
           <NumberFormat
             className={cx('input')}
             thousandSeparator
             decimalScale={6}
-            type="text"
+            isAllowed={(values) => {
+              const { floatValue } = values;
+              return floatValue === undefined || (floatValue >= 0 && floatValue <= 100);
```

Comment:
I think writing !floatingValue instead of floatingValue === undefined is shorter while doing the same work. What do you think?

Reply:
> I think writing !floatingValue instead of floatingValue === undefined is shorter while doing the same work. What do you think?
Yes! I agree with you, in this case it can work well: https://github.com/oraichain/oraiswap-frontend/pull/409/commits/46f1da8e13ae03c74c31d5ac4f81f71152ea6084